package ExceptionHandling;

// exception can be generated by method

public class ExcTest {

	// generate exception
	static void genException() {
		int nums[] = new int[4];
		
		System.out.println("Before exception is generated.");
		
		// Here exception is generating
		// in connection with arrays applying abroad
		nums[7] = 10;
		System.out.println("This won't be displayed!");
		
	}
}
